modules/iam/src/test/scala/ch/epfl/bluebrain/nexus/commons/iam/MockedIAMClient.scala

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.bluebrain.nexus.commons.iam
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpMessage.DiscardedEntity
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.stream.Materializer
+import akka.util.ByteString
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient.UntypedHttpClient
+import ch.epfl.bluebrain.nexus.commons.iam.acls.Permission._
+import ch.epfl.bluebrain.nexus.commons.iam.acls._
+import ch.epfl.bluebrain.nexus.commons.iam.auth.{AuthenticatedUser, User}
+import ch.epfl.bluebrain.nexus.commons.iam.identity.Identity.{AuthenticatedRef, GroupRef, UserRef}
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.syntax._
+
+import scala.concurrent.Future
+
+trait MockedIAMClient {
```

Comment:
consider using mockito instead since it was already introduced to the project (see iam)